perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -682,6 +682,20 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 **
```

Comment:
I'm not sure how our perl parser will handle this. Please make it a single ``*``.

Reply:
Removed.